packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -1213,8 +1217,12 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(CustomChoiceGroupMi
    */
   __setComboboxDisabledAndReadOnly() {
     if (this._comboboxNode) {
-      this._comboboxNode.setAttribute('disabled', `${this.disabled}`);
-      this._comboboxNode.setAttribute('readonly', `${this.readOnly}`);
+      this._comboboxNode.toggleAttribute('disabled', this.disabled);
```

Comment:
```suggestion
      // Since `._comboboxNode` can either be `<div "role=combobox">` or `<input>`,
      // we need to cater for both scenarios (aria for semantics, "native attr" for operability)
      this._comboboxNode.toggleAttribute('disabled', this.disabled);
      this._comboboxNode.setAttribute('aria-disabled', `${this.disabled}`);
```